ecommerce/admin.py

```diff
@@ -25,14 +25,21 @@ class LineAdmin(admin.ModelAdmin):
     """Admin for Line"""
     model = Line
 
-    readonly_fields = get_field_names(Line)
+    readonly_fields = [name for name in get_field_names(Line) if name != 'course_key']
```

Comment:
That's correct. We use the same pattern elsewhere in this admin where only one field is editable.